alerter.cpp

```diff
@@ -24,8 +28,10 @@ void alertInCelcius(float farenheit) {
 }
 
 int main() {
-    alertInCelcius(400.5);
-    alertInCelcius(303.6);
+    alertInCelcius(303.6); // OK
+    assert(alertFailureCount == 0);

```

Comment:
Test code and production code needs to be maintained separately in different files for readability and maintainability.